### PR TITLE
Feature sum

### PR DESCRIPTION
**Pull request by pboserman**
X-cell column sum functionality added.

### CODE REVIEW

File: client/js/array-util.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (speschellk) [inline client/js/array-util.js:1]: Set your text editor to use 2-space tabs instead of 4-space tabs and to “convert indentations to spaces”; ask me if you have any trouble. This is key for readability, especially when you checked your code into a central repo that other devs will work off of.

File: client/js/test/sum-util.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const { getSumFromArray } = require('../sum-util');
```
Comment (speschellk) [inline client/js/test/sum-util.test.js:1]: Add a few more tests that test the functionality of your new feature. Especially test against non-happy paths and edge cases. For instance, “it correctly generates sum when the column includes non-number inputs”.

File: public/js/bundle.js [hunk 3]
```diff
@@ -7,7 +7,8 @@ const tableView = new TableView(model);
 tableView.init();
```
Comment (speschellk) [inline public/js/bundle.js:7]: Your bundle.js file should not be checked into git or GitHub, since it's a big ugly mess of a file that production environments (Heroku) will generate for you anyway. This is why we put it in the .gitignore file at the beginning of development.

Delete from GitHub and don't commit any modifications to bundle.js in git.

File: client/js/test/table-view.test.js [hunk 4]
```diff
@@ -10,6 +10,36 @@ describe('table-view', () => {
         document.documentElement.innerHTML = html;
```
Comment (speschellk) [inline client/js/test/table-view.test.js:10]: Same thing here. What are edge cases and non-happy paths that you can anticipate a user getting into, that you can test for here?
